Homework4/Homework4/Validators.cs

```diff
@@ -0,0 +1,37 @@
+using Homework4.Interfaces;
+
+namespace Homework4;
+
+public class Validator<T>
+{
+    private readonly List<IValidator<T>> _validators;
+
+    public Validator()
+    {
+        _validators = new List<IValidator<T>>();
+    }
+
+    public ValidationResult Validate(T model)
+    {
+        var result = new ValidationResult();
+
+        foreach (var validator in _validators)
+        {
+            try
+            {
+                validator.Validate(model);
```

Comment:
IValidator<T>.Validate method returns a bool value. Where do you use it?

Reply:
Fixed